openedx/core/djangoapps/user_api/tests/test_views.py

```diff
@@ -861,7 +850,7 @@ def test_register_form_default_fields(self):
                 u"name": u"username",
                 u"type": u"text",
                 u"required": True,
-                u"label": u"Username",
+                u"label": u"Public username",
```

Comment:
Were we asked to change this field's label back to "Public username," or was the label an artifact from the current registration form which made it into the new logistration comps? We've used "Public username" to describe this field in the past, but I thought we we're moving to just using "Username" as part of logistration.